charts/incubator/checkmk/values.yaml

```diff
@@ -0,0 +1,44 @@
+image:
+  repository: checkmk/check-mk-raw
+  pullPolicy: IfNotPresent
+  tag: 2.1.0-latest
+
+securityContext:
+  readOnlyRootFilesystem: false
+  runAsNonRoot: false
+
+podSecurityContext:
+  runAsUser: 0
+  runAsGroup: 0
+
+service:
+  main:
+    ports:
+      main:
+        protocol: HTTP
+        targetPort: 5000
+        port: 5000
+  receiver:
+    ports:
+      main:
+        protocol: HTTP
+        targetPort: 8000
+        port: 8000
+
+probes:
+  liveness:
+    path: "/cmk/check_mk"
+
+  readiness:
+    path: "/cmk/check_mk"
+
+  startup:
+    path: "/cmk/check_mk"
+
+persistence:
+  config:
+    enabled: true
+    mountPath: "/omd/sites"
+
```

Comment:
```suggestion
  cmk-temp:
    enabled: true
    type: emptyDir
    mountPath: /opt/omd/sites/cmk/tmp
```

Reply:
Yeah I spotted that but it also wanted a Ramdisk. I assume emptyDir achieves the same thing? Just so I know for the future.